test/data/3bus/af.csv

```diff
@@ -1,4 +1,4 @@
 ﻿area,subarea,genfuel,gentype,year,status,h1,h2,h3,h4,h5,h6,h7,h8
-nation,narnia,solar,,,1,0.5,0.5,0.5,0.0,0.5,0.5,0.5,0.0
-nation,narnia,solar,,y2040,1,0.5,0.6,0.6,0.0,0.5,0.6,0.6,0.0
-nation,archenland,solar,,,1,0.4,0.4,0.4,0.0,0.4,0.4,0.4,0.0
\ No newline at end of file
+bus_nation,narnia,solar,,,1,0.5,0.5,0.5,0.0,0.5,0.5,0.5,0.0
```

Comment:
what's the reason for changing this? I think it's fine to use bus_nation but couldn't you still use nation as well since `af` matters more for where the generator is than where the bus is?

Reply:
Yes, in the RFF inputs it will not get changed to `bus_nation`.  However, for our narnia grid, the generators do not have their own nation so they only get a `bus_nation`.